src/main/steps/c100-rebuild/miam/domestic-abuse/content.ts

```diff
@@ -0,0 +1,428 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { atLeastOneFieldIsChecked } from '../../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  section: 'Miam section',
```

Comment:
I am not sure if this is being used to render. Can we please remove if not being used ?